docs/100.yaml

```diff
@@ -411,6 +411,9 @@
   author : Bolton Bailey, Patrick Stevens
 99:
   title  : Buffon Needle Problem
+  links  :
+    mathlib archive: https://github.com/leanprover-community/mathlib4/blob/master/Archive/Wiedijk100Theorems/BuffonsNeedle.lean
```

Comment:
```suggestion
  decl   : buffon_short buffon_long
```

Reply:
Should I use `BuffonNeedle.buffon_short` and `BuffonNeedle.buffon_long`?

Reply:
Yes, declaration names in the yaml files need to be fully qualified

Reply:
Hmm, looks like there are [build errors](https://github.com/leanprover-community/mathlib4/actions/runs/8362867760/job/22894523493?pr=10189). I wonder if `100.yaml` doesn't have visibility into the files imported by `Archive.lean`?